test/Main.hs

```diff
@@ -31,7 +29,7 @@ genArrayW8Large = do
     let minArr = 1024 * 10
         maxArr = 1024 * 100
     arrS <- choose (minArr, maxArr)
-    Array.fromList <$> vectorOf arrS (elements [0,1])
+    Array.fromList <$> vectorOf arrS (frequency [(9, pure 0), (1, pure 1)])
```

Comment:
I meant to have separate tests for less compressible and more compressible data. We should keep the old test as it is add another case.